### PR TITLE
[test] Fix CI failure caused by legacy DEFAULT_KAFKA_OPERATION_TIMEOUT_MS

### PR DESCRIPTION
**Pull request by haoxu07**
## Summary, imperative, start upper case, don't end with a period

There are some code changed the naming of DEFAULT_KAFKA_OPERATION_TIMEOUT_MS. For this test, it seems the topic manager is not needed.

## How was this PR tested?

CI 
## Does this PR introduce any user-facing changes?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

**Review by sushantmane — APPROVED** [review]
Thanks!

File: internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestHybrid.java [hunk 1]
```diff
@@ -294,16 +294,7 @@ public void testHybridSplitBrainIssue() {
...
-        TopicManager topicManager =
-            IntegrationTestPushUtils
-                .getTopicManagerRepo(
-                    DEFAULT_KAFKA_OPERATION_TIMEOUT_MS,
-                    100,
-                    0l,
-                    venice.getPubSubBrokerWrapper(),
-                    venice.getPubSubTopicRepository())
-                .getTopicManager()) {
+            new ControllerClient(venice.getClusterName(), parentController.getControllerUrl());) {
```
Comment (majisourav99) [inline internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestHybrid.java:297]: nit: can we fix the extra `;)` ?

Comment (haoxu07) [reply internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestHybrid.java:297]: Sure, let me fix that.

**Review by majisourav99 — APPROVED** [review]
Thanks!

**Pull request merged**